src/modules/customers/dto/get-customers-query.dto.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { CUSTOMERS, UKEFID } from '@ukef/constants';
+import { ENUMS } from '@ukef/constants/enums';
+import { FallbackToLegacyDataEnum } from '@ukef/constants/enums/fallbackToLegacyData';
+import { IsEnum, IsOptional, Matches, MaxLength, MinLength } from 'class-validator';
+export class GetCustomersQueryDto {
+  @IsOptional()
+  @ApiProperty({
+    required: false,
+    example: CUSTOMERS.EXAMPLES.COMPANYREG,
+    description: 'The company registration number (in the country where the customer is registered) of the customer to search for.',
+  })
+  @MinLength(8)
+  @MaxLength(10)
+  public companyreg?: string;
```

Comment:
In the ticket, we spelt this as `companyReg` instead - did we want to change that? (I prefer `companyReg` myself as it matches the response body)

Reply:
Agree, changed companyReg, but in response it is companyRegNo